readthedocs/core/static-src/core/js/doc-embed/sponsorship.js

```diff
@@ -71,7 +71,7 @@ function create_ad_placement() {
         // Assumes the ad would be ~200px high
         element = $("<div />").appendTo(selector);
         offset = element.offset();
-        if (!offset || (offset.top - $(window).scrollTop() + 200) > $(window).height()) {
+        if (!offset || (offset.top - window.scrollTop + 200) > window.offsetHeight) {
```

Comment:
Doesn't look like `window.scrollTop` or `window.offsetHeight` exist on the window element. Looks like we can use

- https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollY
- https://developer.mozilla.org/en-US/docs/Web/API/Window/innerHeight

Reply:
Thanks, I've changed this in bcde94a3ea476fa82e3c09e4be5a3315cdad1e7b.

It's probably not a good idea to write the code without any testing whatsoever ...